### PR TITLE
Issue #191: Set container cpu limit in CRI job environement

### PR DESCRIPTION
**Pull request by koct9i**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

### CODE REVIEW

File: yt/yt/server/node/exec_node/job.cpp [hunk 1]
```diff
@@ -2338,6 +2338,13 @@ TJobProxyInternalConfigPtr TJob::CreateConfig()
         proxyConfig->MemoryTracker->UseSMapsMemoryTracker = false;
     }
 
+    if (UserJobSpec_ && UserJobSpec_->set_container_cpu_limit()) {
+        proxyConfig->ContainerCpuLimit = UserJobSpec_->container_cpu_limit();
+        if (*proxyConfig->ContainerCpuLimit < 0) {
```
[the remark this answers is not in the document]
Comment (koct9i) [reply yt/yt/server/node/exec_node/job.cpp:2343]: I've implemented the same logic as job-proxy has for porto job-environment:

https://github.com/ytsaurus/ytsaurus/blob/c6039f5f41cfe1eb2467ce8fbe12e3654e88f491/yt/yt/server/job_proxy/job_proxy.cpp#L820

File: yt/yt/server/lib/job_proxy/config.cpp [hunk 2]
```diff
@@ -217,6 +217,8 @@ void TJobProxyInternalConfig::Register(TRegistrar registrar)
     registrar.Parameter("original_cluster_connection", &TThis::OriginalClusterConnection)
         .Default();
 
+    registrar.Parameter("container_cpu_limit", &TThis::ContainerCpuLimit);
```
Comment (gritukan) [inline yt/yt/server/lib/job_proxy/config.cpp:220]: Shouldn't we add `.Default()` here?

File: yt/yt/server/node/exec_node/job.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (gritukan) [reply yt/yt/server/node/exec_node/job.cpp:2343]: I'm pretty sure that the negation of > 0 here is == 0, not < 0

**Review by gritukan — CHANGES_REQUESTED** [review]

File: yt/yt/server/node/exec_node/job.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (koct9i) [reply yt/yt/server/node/exec_node/job.cpp]: my bad, should be ( <= 0)

**Comment by yalekseev** [comment]
Shipped!

**Pull request closed without merging**